proto/resource_manager.proto

```diff
@@ -231,7 +231,10 @@ message RunawaySettings {
 }
 
 message BackgroundSettings {
-  repeated string job_types = 1; 
+  // background task types.
+  repeated string job_types = 1;
+  // the percentage limit of total resource(cpu/io) that background tasks can use.
+  uint64 utilization_limit = 2;
```

Comment:
Should we use separated limits for IO and CPU?

Reply:
Currently not, if both cpu and io are limited, the lower will take effect. BTW, in most cases, IO is not limited because it depends on the total IO quota which is not set by default.